packages/blocks/src/api/test/factory.js

```diff
@@ -1554,12 +1555,34 @@ describe( 'block factory', () => {
 	} );
 
 	describe( 'isContainerGroupBlock', () => {
-		it( 'should return true when passed block name matches "core/group"', () => {
-			expect( isContainerGroupBlock( 'core/group' ) ).toBe( true );
+		beforeEach( () => {
```

Comment:
So parent `describe` unregisters all blocks for each test. That was confusing at first but it's perfectly valid 👍

Reply:
Yep. I'm adding tests to an existing suite so didn't want to disrupt.